vir/test.cpp

```diff
@@ -155,6 +241,7 @@ static_assert(std::same_as<vir::as_tuple_t<Line>,
 static_assert(std::same_as<vir::simdize<Line>,
 			   vir::simd_tuple<Line, V<float>::size()>>);
 
+#if VIR_HAVE_SIMD_IOTA
```

Comment:
This looks unrelated as well?

Reply:
Yes. A drive-by clean up that I don't believe warrants its own commit.